internal/pkg/aws/acm/acm.go

```diff
@@ -66,23 +71,31 @@ func (a *ACM) ValidateCertAliases(aliases []string, certs []string) error {
 	}
 	for _, alias := range aliases {
 		if !validAliases[alias] {
+			for certARN, domains := range AllowedDomainsCert {
+				log.Infoln(fmt.Sprintf("Your imported Certificate '%s' protects the following domains:", certARN))
+				for _, domain := range domains {
+					log.Infoln(domain)
+				}
+			}
 			return fmt.Errorf("%s is not a valid domain against %s", alias, strings.Join(certs, ","))
 		}
 	}
 	return nil
 }
 
-func (a *ACM) findValidAliasesAgainstCert(ctx context.Context, aliases []string, cert string) ([]string, error) {
+func (a *ACM) findValidAliasesAndAllowedDomainsAgainstCert(ctx context.Context, aliases []string, cert string) ([]string, map[string][]string, error) {
 	resp, err := a.client.DescribeCertificateWithContext(ctx, &acm.DescribeCertificateInput{
 		CertificateArn: aws.String(cert),
 	})
 	if err != nil {
-		return nil, fmt.Errorf("describe certificate %s: %w", cert, err)
+		return nil, nil, fmt.Errorf("describe certificate %s: %w", cert, err)
 	}
 	domainSet := make(map[string]bool)
+	domainsperCert := make(map[string][]string)
```

Comment:
I might be missing something - it seems like the only key would be `cert`. If this is indeed the case, why do we need a `map`?

Reply:
Just changed the map to [ ]string